### ISSUE TITLE
"New calendar" submit icon is not focusable

### ISSUE THREAD

**Issue by ChristophWurst** [nextcloud/calendar#4125]
### Steps to reproduce

1. Open the app
2. Create a new calendar via keyboard

### Expected behavior

The submit icon is focusable

### Actual behaviour

The submit icon is not focusable

### Calendar app version

3.x

### Additional info

Originally reported in https://github.com/nextcloud/calendar/issues/3540.

**Comment by JuliaKirschenheuter** [comment]
tested and it works
